docs/topic-guides/model-definitions/index.txt

```diff
@@ -0,0 +1,43 @@
+.. _model-definitions:
+
+Model Definitions
```

Comment:
blocking: Now that I'm getting to here, it's feeling slightly redundant with the index. I'd err on the side of doing a much shorter intro in index and linking here instead.

Reply:
I guess I was envisioning that you would say "First start with Quick Start, then read Model Definitions for a deeper understanding" or something. But this is mostly my gut feeling so I won't block here.